fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/common/StoreTimer.java

```diff
@@ -376,22 +378,32 @@ public interface Wait extends StoreTimer.Event {
 
     /**
      * {@link Event}s that only count occurrences or total size.
-     * There is no meaningful time duration associated with these events.
+     * There is no meaningful cumulative value associated with these events.
      */
     public interface Count extends StoreTimer.Event {
         /**
          * Get whether the count value is actually a size in bytes.
          *
          * @return {@code true} if the count value is actually a size in bytes
          */
-        boolean isSize();
+        default boolean isSize() {
+            return false;
+        }
     }
 
     /**
-     * Contains the number of occurrences and cummulative time spent on an associated {@link StoreTimer.Event}.
+     * {@link Event}s that only count the number of occurrences of a read/write operation and also the size read/written.
+     * There is no meaningful time duration associated with these events.
```

Comment:
As implemented, isn't the time going to return whatever the size is?
That seems somewhat less than ideal to have it be ambiguous.

Reply:
Beneath the scene, yes. However, the converse is not true because for getting and recording the size, the consumer would require explicitly a SizeEvent. If it's really pressing, we can separate the concerns.

Reply:
I think there's not really a good way to adjust.
If it weren't for backwards compatibility, I would say that we should have `TimeEvent` and `SizeEvent` as siblings in a hierarchy. rather than having `SizeEvent` extending `Event` which implies time.